firmware/arduino/chessboard/uart.ino

```diff
@@ -70,7 +95,14 @@ void serialEvent2()
                 // Sends acknowledgement
                 sendMessageToPi(currentState, buffer[5], errorCode);
 
+                movementFlag = true;
                 makeMove(buffer);
+
+                // Tell Pi that the chess timer button was pressed
+                if (transmitFlag)
+                    sendMessageToPi(END_TURN, 0, buffer[5]);
+                movementFlag = false;
```

Comment:
Please add a space between lines 103 and 104

Reply:
I think it's grouped appropriately now that movementFlag is with the makeMove() function call.

Reply:
```
movementFlag = true;
makeMove(buffer);
movementFlag = false;

// Tell Pi that the chess timer button was pressed
if (transmitFlag)
    sendMessageToPi(END_TURN, 0, buffer[5]);
transmitFlag = false;
```